src/core/server/deprecations/README.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: kibUsageCollectionPlugin
+slug: /kibana-dev-docs/services/deprecations-service
+title: Core Deprecations service
+summary: The Deprecations service helps surface deprecated configs and features for plugins to our users
+date: 2021-05-03
+tags: ['kibana','dev', 'contributor', 'api docs']
+---
+
+# Core Deprecations service
+This guide is written in the format of a FAQ to help you get started using the deprecations service.
+For more details on the service contract we've documented all the service apis and properties under the deprecation service docs generated by in core.
+
+## What is the deprecations service?
+The deprecations service provides a way for the Kibana platform to communicate deprecated features and configs with its users. These deprecations are only communicated if the deployment is using these features, allowing for a user tailored experience for upgrading the stack version.
+
+## Where are deprecations displayed to users?
+The Upgrade Assistant (UA) in Kibana is consuming the core deprecations service in order to surface plugin deprecations. Each deprecation should have manual steps that we display in the UI to fix the deprecation or an API to automatically resolve it.
+
+To check your plugin deprecations go to the UA interface and click on `view deprecations` under the kibana section.
+
+## How do I use this service for deprecated plugin configurations?
+The deprecations service automatically hooks deprecated configs with the deprecations service. 
+
+All the config deprecation functions (`unused`, `unusedFromRoot`, `rename`, `renameFromRoot`) accept an optional parameter to customize the deprecation details.
```

Comment:
What is the difference between unused and unusedFromRoot? Maybe explain when to use both options. Same with rename.

Reply:
Added a link to the docs for these functions :+1: thanks!